### ISSUE TITLE
Date not displaying in Android footer bar

### ISSUE THREAD

**Issue by whitemx** [unplugged/unplugged-controls#1]
This is a known issue with Unplugged which has been reported by Rich. In the footer bar the last replicated date should be shown in dd/mmm/yyyy hh:mm:ss format but in Android it shows as number of milliseconds since 01/01/1970.

**Comment by whitemx** [comment]
This will be fixed by the Unplugged dev, team, just logging here for completeness sake.

**Comment by sives** [comment]
For the production release of the Controls, I would like to make this "time since last sync" a bit more user-friendly.
Even if it means we need to write a bit more code in the footer bar (Matt - happy to incur the extra cost)

This is the sort of "time ago" info that I would prefer:

Last sync 1m ago (if less than one minute)
Last sync 15m ago
Last sync 3h 17m ago
Last sync 1d 6h ago

When more than 10 days has elapsed since the last sync, just display the days:
Last sync 225 days ago

**Comment by whitemx** [comment]
Steve

This is feedback from Rich...

---

It looks like that's one that slipped through among all the camera mayhem.  It is still different between platforms; it's a string on iOS ("12/04/12 05:12:11 PM"), and a number of mills since 1/1/1970 on Android.

## We'll get this fixed after 2.2 goes gold so we can include it in a 2.2.1 (if a maintenance release is needed for any other reasons) and definitely for 2.3.  It's a little dicey because it will break existing code, but there's no way around that, I fear.

Basically it looks like this will not be possible in the current release.

Matt

**Comment by RichSharpe** [comment]
Checked with Unplugged dev team. This will be addressed in 2.3. There will be a new var ro.last.sync that will be compatible on both OS's
